src/outro.js

```diff
@@ -1,2 +1,13 @@
+for (var key in MathQuill) (function(key, val) {
+  if (preInterVerMathQuill[key]) return; // already set .noConflict
+  if (typeof val === 'function') {
+    preInterVerMathQuill[key] = function() {
+      insistOnInterVer();
```

Comment:
This is failing in Mac Chrome 45.0.2454.85 with:
TypeError: Cannot set property 'prototype' of undefined(anonymous function)

The reason for this is `typeof preInterVerMathQuill === 'number'`.  While Chrome does not throw an error, when you assign a property to a number, you cannot actually access that property; it will always be undefined.

The following a Chrome console I tested, each set is my input plus Chrome's output:
```javascript
var test = 1;
undefined

test["special"] = 'something awesome'
"something awesome"

test["special"]
undefined
```

Whatever you're using to store the interface version needs to be a real Object.